docs/examples/example_oosm_algorithm.py

```diff
@@ -0,0 +1,405 @@
+                    state_vector=sensor_2_mm.function(pseudo_state, noise=False),
+                    groundtruth_path=detections_2.groundtruth_path,
+                    measurement_model=sensor_2_mm,
+                    timestamp=pseudo_state.timestamp)
+
+            else:
+                pseudo_meas = Clutter(
+                    state_vector=sensor_2_mm.function(pseudo_state, noise=False),
+                    measurement_model=sensor_2_mm,
+                    timestamp=pseudo_state.timestamp)
+
+            # Add the pseudo measurements
+            detections = [detections_1, pseudo_meas]
+        else:
+            # If the there is no delay
+            detections = [detections_1, detections_2]
+
+        # perform the association
+        associations = data_associator.associate(oosm_tracks, detections, timestamps[k])
+
+        # loop over the hypotheses
+        for track, hypotheses in associations.items():
+            for hypothesis in hypotheses:
+                if not hypothesis:
+                    # if it is not an hypothesis, use the prediction
+                    track.append(hypothesis.prediction)
+                else:
+                    post = updater.update(hypothesis)
+                    track.append(post)
+
+# %%
+# Run the tracker while ignoring OOSM measurements
+#
+
+for k in range(len(scan_s1)):
+    for detections_1, detections_2 in zip(scan_s1[k], scan_s2[k]):
+        # Include only the In-sequence measurements
+        if timestamps[k] == detections_2.timestamp:
+            detections = [detections_1, detections_2]
+
+            associations = data_associator.associate(noOsm_tracks, detections, timestamps[k])
+
+            for track, hypotheses in associations.items():
+                for hyp in hypotheses:
+                    if not hyp:
+                        track.append(hyp.prediction)
+                    else:
+                        post = updater.update(hyp)
+                        track.append(post)
+
+
+# %%
+# 4. Run the comparison between the resulting tracks and plot the results.
+# ------------------------------------------------------------------------
+# We have obtained the final tracks from the detections and we have two sets of
+# tracks, one with OOSM and one without. We now can visualise the results
+# and evaluate the track-to-track accuracy using the OSPA metric tool.
+#
+
+from stonesoup.plotter import AnimatedPlotterly
+plotter = AnimatedPlotterly(timesteps=timestamps)
+plotter.plot_ground_truths(truths, [0, 2])
+plotter.plot_measurements(scan_s1, [0, 2], measurements_label='scan1', measurement_model=sensor_1_mm)
+plotter.plot_measurements(scan_s2, [0, 2], measurements_label='scan2', measurement_model=sensor_1_mm)
+plotter.plot_tracks(oosm_tracks, [0, 2], track_label='OOSM Tracks',
+                    line= dict(color='orange'))
+plotter.plot_tracks(noOsm_tracks, [0, 2], track_label='no-OOSM Tracks',
+                    line= dict(color='red'))
+plotter.fig
```

Comment:
The `TrueDetection`s plotted in your output (blue points) look pretty noisy and a long way from the ground truth towards the end of the simulation. Ignore me if that's intentional, but might be worth adjusting the noise in your measurement model if not.